.circleci/config.yml

```diff
@@ -20,9 +32,21 @@ jobs:
       - run: sudo apt update && sudo apt install -y graphviz
       - checkout
       - run: virtualenv venv
-      - run: source venv/bin/activate && make installdeps
+      - run: |
+          source venv/bin/activate
+          python setup.py sdist
+          tar -zxvf "dist/featuretools-$(python setup.py --version).tar.gz"
+      - run: |
+          source venv/bin/activate
+          FEATURETOOLS_VERSION=$(python setup.py --version)
+          pip install -e "featuretools-${FEATURETOOLS_VERSION}/"
+          pip install -r "featuretools-${FEATURETOOLS_VERSION}/test-requirements.txt"
+          pip install -r dev-requirements.txt
```

Comment:
Dev requirements installs test-requirements.
So it seems like the above 2 lines are redundant.
https://github.com/Featuretools/featuretools/blob/master/dev-requirements.txt#L1